docs/guides/configure-rules.md

```diff
@@ -0,0 +1,155 @@
+
+The `subject` is the `version` property of the `info` object; Info is a [recognized node type](https://redocly.com/docs/openapi-visual-reference/openapi-node-types/). Using the `pattern` assertion, describe what's allowed with [regular expression syntax](https://en.wikipedia.org/wiki/Regular_expression). Finally, adding the `message` ensures clear information is conveyed if the rule isn't satisfied.
+
+Using the supplied assertion features, you can extend the built-in rules in Redocly and describe the success criteria for APIs in your organization.
+
+## Set up per-API configuration
+
+Not all APIs are made equal. Perhaps your newsletter signup microservice is held to a different standard than the finance-related APIs in use in your organisation. Perhaps one API can't be changed, but newer versions can be checked against stricter rules. Whatever the reason, per-API configuration can be useful in these situations.
+
+You can configure linting differently for multiple APIs in a Redocly configuration file. Here's an example with three APIs defined, and different rules applied for each:
+
+```yaml
+extends:
+  - recommended
+
+apis:
+  partnerApi@1:
+    root: partners.yaml
+    rules:
+      no-server-trailing-slash: off
+      operation-operationId: off
+
+  newsletter@3:
+    root: /absolute/path/to/folder/openapi.yaml
+    extends:
+      - minimal
+    rules:
+      no-server-trailing-slash: off
+
+  UserApi@1:
+    root: users.yaml
+    rules:
+      no-server-trailing-slash: off
+      operation-operationId: warn
+      assert/version-semver:
+        subject:
+          type: Info
+          property: version
+        assertions:
+          pattern: /1.[0-9]\.[0-9]/
+        message: API version must be in SemVer format, no major version release
+```
+
+There's a few things going on in the example, but let's look at each feature in turn:
+* The first section configures `recommended` as the default ruleset for all APIs. The "newsletter" API overrides this by declaring `minimal` for itself, but the others inherit the top-level setting.
+* No rules are defined at the top level, but since every API sets the `no-server-trailing-slash` rule to "off", this could be set at the top level.
+* Each API adds (or removes) the rules that fit their use case, including using the `version-semver` assertion.
+
+Configuring per-API means that there doesn't have to be compromise for the lowest standard that all APIs can meet. Especially when you are working on improving your APIs or API descriptions, setting the desired ruleset and adding exceptions until the API meets all requirements in full is a good way to ensure standards only improve.
+
+## Next steps
+
+* Check out the detailed reference documentation for the [built-in rules](../rules/builtin-rules.md) and [configurable rules](../rules/configurable-rules.md).
+* If nothing there meets your needs, you can create your own rules by creating [custom plugins](../resources/custom-plugins.md) with JavaScript.
```

Comment:
I think just "plugins" is actually a really good idea, since we don't publish our own (by definition, ours are inside the product!). Also the rules aren't that custom, other than the custom function ones, so I think configurable rules works as well as any of the other options.

(Plugins docs need some work but in a separate PR!)